code/modules/admin/verbs/randomverbs.dm

```diff
@@ -9,7 +9,13 @@
 	if(confirm != "Yes")
 		return
 
-	for(var/obj/item/W in M)
+	var/list/diff = list()
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		diff = H.organs ^ H.internal_organs //Ignore organs and stuff
+
+	var/list/dropstuff = difflist(M.contents, diff)
+	for(var/obj/item/W in dropstuff)
```

Comment:
Well you'll be surprised to know that drop everything will just dump any and all items inside of you indiscriminately.
Try it live.